.github/workflows/ci.yaml

```diff
@@ -49,7 +49,10 @@ jobs:
       with:
         distribution: zulu
         java-version: ${{matrix.java}}
-    - uses: google-github-actions/auth@35b0e87d162680511bf346c299f71c9c5c379033 # v1.1.1
+    - name: Authenticate to Google Cloud
+      # only needed for Flakybot on periodic (schedule) and continuous (push) events
+      if: ${{ github.event_name == 'schedule' || github.event_name == 'push' }}
+      uses: google-github-actions/auth@35b0e87d162680511bf346c299f71c9c5c379033 # v1.1.1
```

Comment:
fyi: this is more efficient and will result in faster pull request builds

Reply:
@enocom looks like our Java unit tests require valid google auth credentials, this seems off to me...?

Reply:
moved to separate issue: #314